src/WebJobs.Script.WebHost/Properties/Resources.resx

```diff
@@ -259,6 +259,21 @@
     ]
 }</value>
   </data>
+  <data name="SwaggerFileDeleteError" xml:space="preserve">
+    <value>Error deleting swagger document location at:{0}</value>
```

Comment:
These strings don't read right. Do you mean "Error deleting swagger document: {0}"